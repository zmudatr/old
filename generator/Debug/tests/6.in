USTAW F epevg9iU
USTAW O 7/Z3I+3t
DOKLEJ F O
WYPISZ F
WYPISZ O
ZERUJ F
ZERUJ O
USTAW P qRS6H
USTAW B wQ+
DOKLEJ P B
WYPISZ P
WYPISZ B
ZERUJ P
ZERUJ B
USTAW B 7aoX
USTAW E g2raK
DOKLEJ B E
WYPISZ B
WYPISZ E
ZERUJ B
ZERUJ E
USTAW D Hd5vYLw*b
USTAW G K8Wy
DOKLEJ D G
WYPISZ D
WYPISZ G
ZERUJ D
ZERUJ G
USTAW K kSu
USTAW T Q
DOKLEJ K T
WYPISZ K
WYPISZ T
ZERUJ K
ZERUJ T
USTAW E NPDcT75HS
USTAW Z eWje4A9C
DOKLEJ E Z
WYPISZ E
WYPISZ Z
ZERUJ E
ZERUJ Z
USTAW U XPGZ_0HI8
USTAW A za5eiP
DOKLEJ U A
WYPISZ U
WYPISZ A
ZERUJ U
ZERUJ A
USTAW H Fnk-Qr1c
USTAW U P1+Bj
DOKLEJ H U
WYPISZ H
WYPISZ U
ZERUJ H
ZERUJ U
USTAW U oYaA
USTAW O AKzdF
DOKLEJ U O
WYPISZ U
WYPISZ O
ZERUJ U
ZERUJ O
USTAW X fCT
USTAW I 5TszCjKl/p
DOKLEJ X I
WYPISZ X
WYPISZ I
ZERUJ X
ZERUJ I
USTAW A gdIeLJsk
USTAW Q 4uqv*pnOGN
DOKLEJ A Q
WYPISZ A
WYPISZ Q
ZERUJ A
ZERUJ Q
USTAW G O1JxOeFWM+
USTAW B 4Sma2o
DOKLEJ G B
WYPISZ G
WYPISZ B
ZERUJ G
ZERUJ B
USTAW K AMHp7hiO0z
USTAW N lp
DOKLEJ K N
WYPISZ K
WYPISZ N
ZERUJ K
ZERUJ N
USTAW W WY_BUy+Si
USTAW G G
DOKLEJ W G
WYPISZ W
WYPISZ G
ZERUJ W
ZERUJ G
USTAW M K
USTAW R 9hWG/b
DOKLEJ M R
WYPISZ M
WYPISZ R
ZERUJ M
ZERUJ R
USTAW Z mN
USTAW P 9fl
DOKLEJ Z P
WYPISZ Z
WYPISZ P
ZERUJ Z
ZERUJ P
USTAW K Q/0
USTAW B XySyWpIz
DOKLEJ K B
WYPISZ K
WYPISZ B
ZERUJ K
ZERUJ B
USTAW Z cZ
USTAW N kn
DOKLEJ Z N
WYPISZ Z
WYPISZ N
ZERUJ Z
ZERUJ N
USTAW B 7sLoTo
USTAW K xXFW
DOKLEJ B K
WYPISZ B
WYPISZ K
ZERUJ B
ZERUJ K
USTAW H q94le
USTAW W 4xxKYG_J48
DOKLEJ H W
WYPISZ H
WYPISZ W
ZERUJ H
ZERUJ W
