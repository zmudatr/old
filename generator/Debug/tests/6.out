epevg9iU7/Z3I+3t

qRS6HwQ+

7aoXg2raK

Hd5vYLw*bK8Wy

kSuQ

NPDcT75HSeWje4A9C

XPGZ_0HI8za5eiP

Fnk-Qr1cP1+Bj

oYaAAKzdF

fCT5TszCjKl/p

gdIeLJsk4uqv*pnOGN

O1JxOeFWM+4Sma2o

AMHp7hiO0zlp

WY_BUy+SiG

K9hWG/b

mN9fl

Q/0XySyWpIz

cZkn

7sLoToxXFW

q94le4xxKYG_J48

